src/providers/ldap/sdap_async_nested_groups.c

```diff
@@ -2452,6 +2489,7 @@ sdap_nested_group_deref_direct_process(struct tevent_req *subreq)
     ret = EOK;
 
 done:
+    if (members_set) hash_destroy(members_set);
```

Comment:
`hash_destroy()` can handle `NULL`, you only have to be careful if you want to check the return value. If you prefer to keep the `if`, I'd prefer to check for `members_set != NULL`, see above.